### ISSUE TITLE
Auto-expanding 

### ISSUE THREAD

**Issue by mortang2410** [PythonNut/zsh-config#49]
Hi, I found your repo by accident. Great job, I think I have learned quite a few things from it. 
However, I notice that auto-expansion is somewhat overzealous.  For instance, typing 
`bash -c "~` would automatically replace `"~` with just `~`. I would like to remove this auto-expanding capability. Do you think it is possible?

**Comment by PythonNut** [comment]
Yes, this is a problem I've been chasing for a while. It's related to `auto-fu`.
